### ISSUE TITLE
What license should this repo have?

### ISSUE THREAD

**Issue by samreid** [phetsims/smithers#3]
See https://github.com/phetsims/tasks/issues/875

**Comment by jonathanolson** [comment]
It's unlikely things would be reused by other people, but I wouldn't have a problem if people did. I set it up as MIT, should that change?

**Comment by samreid** [comment]
I'm not sure what's best for this repo.  Based on my limited knowledge, I think I'd recommend to make it private with a license like phet-io.  But I'll trust your judgment.

**Comment by jonathanolson** [comment]
@ariel-phet, any preferences for whether this repo should be private?

**Comment by ariel-phet** [comment]
@jonathanolson yeah, I think I would go with private, since this deals with out server side stuff.
